KubeArmor/BPF/system_monitor.c

```diff
@@ -225,12 +237,12 @@ typedef struct __attribute__((__packed__)) sys_context
 } sys_context_t;
 
 #define BPF_MAP(_name, _type, _key_type, _value_type, _max_entries) \
-    struct bpf_map_def SEC("maps") _name = {                        \
-        .type = _type,                                              \
-        .key_size = sizeof(_key_type),                              \
-        .value_size = sizeof(_value_type),                          \
-        .max_entries = _max_entries,                                \
-    };
+    struct {                                                        \
```

Comment:
Can we update our libbpf submodule to latest libbpf version?
Since our code is compatible with v1.0 now.

Ref #1580